CMakeLists.txt

```diff
@@ -89,13 +89,13 @@ set(DATADIR "share")
 endif(WIN32)
 set(SHAREDIR ${DATADIR})
 
-if (NOT APPLE)
-    set(PLUGINDIR "${CMAKE_INSTALL_FULL_LIBDIR}/${APPLICATION_SHORTNAME}/plugins" CACHE STRING "Extra path to look for Qt plugins like for VFS. May be relative to binary.")
-else()
-    # Inside the .app bundle
-    set(PLUGINDIR "../PlugIns" CACHE STRING "Extra path to look for Qt plugins like for VFS. May be relative to binary.")
+if (MIRALL_VERSION_SUFFIX)
```

Comment:
I don't thin that's the right condition.  QA and community are mainly testing rc build and such. So it is best if that did not do any of such change.

Reply:
The only issue here occurs on a dev system, when plugins are loaded from the build folder instead of the install location, this shouldn't be an issue for qa.
But yes we could add an option instead. What do you think?